crates/sui-replay/src/replay.rs

```diff
@@ -52,21 +51,14 @@ use sui_types::storage::get_module_by_id;
 use sui_types::storage::{BackingPackageStore, ChildObjectResolver, ObjectStore, ParentSync};
 use sui_types::sui_system_state::epoch_start_sui_system_state::EpochStartSystemState;
 use sui_types::temporary_store::InnerTemporaryStore;
-use sui_types::transaction::CertifiedTransaction;
-use sui_types::transaction::Transaction;
-use sui_types::transaction::TransactionData;
-use sui_types::transaction::VerifiedTransaction;
-use sui_types::transaction::{InputObjectKind, InputObjects, TransactionKind};
-use sui_types::transaction::{SenderSignedData, TransactionDataAPI};
+use sui_types::transaction::{
+    CertifiedTransaction, InputObjectKind, InputObjects, SenderSignedData, Transaction,
+    TransactionData, TransactionDataAPI, TransactionKind, VerifiedTransaction,
+};
 use sui_types::DEEPBOOK_PACKAGE_ID;
 use tracing::{error, warn};
 
 // TODO: add persistent cache. But perf is good enough already.
-// TODO: handle safe mode
```

Comment:
I didn't see anything safe-mode related in this PR, was this line also intended to be deleted?